### PR TITLE
Stop using case-sensitive status matching

### PR DESCRIPTION
**Pull request by johnclary**
Fixes https://github.com/cityofaustin/atd-data-tech/issues/7677 by using better issue status values across the app.

### CODE REVIEW

File: styles/custom.scss [hunk 1]
```diff
@@ -81,7 +81,7 @@ $theme-colors: (
   }
 }
 
-@import "node_modules/bootstrap/scss/bootstrap";
+@import "~bootstrap/scss/bootstrap";
```
Comment (johnclary) [inline styles/custom.scss:84]: trying to fix this [build error](https://app.netlify.com/sites/atd-product/deploys/6188390a2778b80008032614), which i can't reproduce locally :/

**Comment by johnclary** [comment]
ok, looks like the netlify build issue was related to the `next` version set to `latest`.  ready for review :)

File: package.json [hunk 2]
```diff
@@ -6,8 +6,8 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
-    "bootstrap": "^4.5.3",
-    "next": "latest",
```
Comment (johnclary) [inline package.json:10]: dis was the problem

**Review by amenity — APPROVED** [review]
👍🙏

**Comment by johnclary** [comment]
@amenity unfortunately, this is going to break links that used the project status URL params. E.g.
This: `https://austinmobility.io/projects?status=In%20Progress` is now this: `https://deploy-preview-29--atd-product.netlify.app/projects?status=in_progress`

how bad is that?

**Comment by amenity** [comment]
@johnclary - gtk but nbd. ;P

**Review by mateoclarke — COMMENTED** [review]
👍

**Pull request merged**